parlai/core/torch_ranker_agent.py

```diff
@@ -176,6 +177,45 @@ def __init__(self, opt, shared=None):
                 self.model, device_ids=[self.opt['gpu']], broadcast_buffers=False
             )
 
+    def set_interactive_mode(self, mode, shared=False):
+        if mode:
+            if not shared:
+                # Only print in the non-shared version.
+                print("[" + self.id + ': full interactive mode on.' + ']')
+            self.eval_candidates = 'fixed'
+            self.ignore_bad_candidates = True
+            self.encode_candidate_vecs = True
+            self.fixed_candidates_path = self.opt['fixed_candidates_path']
+            if (self.fixed_candidates_path is None or
+                self.fixed_candidates_path == ''):
+                # Attempt to get a standard candidate set for the given task
+                path = self.get_task_candidates_path()
+                if path:
+                    if not shared:
+                        print("[setting fixed_candidates path to: " + path + " ]")
+                    self.fixed_candidates_path = path
+        else:
+            self.eval_candidates = self.opt['eval_candidates']
+            self.ignore_bad_candidates = self.get('ignore_bad_candidates', False)
+            self.encode_candidate_vecs = self.opt['encode_candidate_vecs']
+            self.fixed_candidates_path = self.opt['fixed_candidates_path']
+
+
+    def get_task_candidates_path(self):
+        path = self.opt['model_file'] + '.cands-' + self.opt['task'] + '.cands'
+        if os.path.isfile(path) and self.opt['fixed_candidate_vecs'] == 'reuse':
+            return path
+        print("[ building candidates file as they do not exist: " + path + ' ]')
```

Comment:
sweet this is nice

Reply:
can we make that print statement a little more loud, in case you provided a filepath and it oculdn't find it?

Reply:
how do i make it loud?  like "**** " ?